tools/NuGet/BuildDynamoZeroTouchLibraryPackage.bat

```diff
@@ -16,10 +16,4 @@ mkdir .\%zt%\tools
 copy %base%\ProtoGeometry.dll .\%zt%\lib\%framework%\ProtoGeometry.dll
```

Comment:
base is defined as bin\AnyCPU\Debug, can you please update this to use Release build.